src/validation.cpp

```diff
@@ -1159,6 +1159,20 @@ CTransactionRef GetTransaction(const CBlockIndex* const block_index, const CTxMe
 {
     LOCK(cs_main);
```

Comment:
Just realized after merge that cs_main is not needed to read from the mempool (only to write to it)